client/package.json

```diff
@@ -2,13 +2,15 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:4000",
   "dependencies": {
     "@testing-library/jest-dom": "^5.13.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
```

Comment:
You could (at some point) move some dependencies to `devDependencies`, if they are not run in production. Like the testing-library.